Notification_Repeat/app/src/main/java/com/example/spanish_notification_repeat/ReminderBroadcast.java

```diff
@@ -0,0 +1,36 @@
+package com.example.spanish_notification_repeat;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+import java.util.Random;
+
+public class ReminderBroadcast extends BroadcastReceiver {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+
+        String[] phrases = {"Buenos días = Good morning.", "Buenas tardes = Good afternoon","Buenas noches = Good evening"
```

Comment:
In good OOP you would create Quotes class and then put all of your quotes logic in there